src/Bicep.Core.UnitTests/Assertions/DiagnosticCollectionExtensions.cs

```diff
@@ -23,6 +24,13 @@ public DiagnosticCollectionAssertions(IEnumerable<Diagnostic> diagnostics)
         {
         }
 
+        public AndConstraint<DiagnosticCollectionAssertions> BeEmpty()
+        {
+            AssertionExtensions.Should(Subject).BeEmpty("contained diagnostics: {0}", string.Join(Environment.NewLine, Subject.Select(d => d.ToString())));
+
+            return new AndConstraint<DiagnosticCollectionAssertions>(this);
+        }
```

Comment:
This seems like a nice improvement. When you think there should be no errors, it will actually tell you what the errors are so you can investigate :) I got tired of having to debug to find out what was wrong.

I would make the same improvement to some of these other methods, but I don't totally know how they work 😆